src/lib/agent/BrowserAgent.prompt.ts

```
@@ -216,75 +145,61 @@ The index parameter refers to the element's position in the page's interactive e
-3. Be concise - just state what you did
-
-Examples:
-- Step: "go to amazon.com and then call done_tool to signal completion"
-  → Navigate to amazon.com, then call done_tool
-  
-- Step: "list tabs and then call done_tool to signal completion"
-  → List tabs, then call done_tool
-
-REMEMBER: If the instruction says to call done_tool, you MUST do it to complete the task.`;
+// Generate prompt for executing TODOs in complex tasks
+export function generateSingleTurnExecutionPrompt(task: string): string {
```

ESLint

[error] 'task' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)